examples/sessions/README.md

```diff
@@ -0,0 +1,14 @@
+# sessions example
+
+This example shows how you might write a simple website using the built-in sessions support.
+
+To scaffold this example, we ran:
+
+```shell
+cargo run -- examples/sessions --templates=on --csrf=on --honeycomb=on --redis=on
+```
+
+To run this, you need redis running on localhost on its default port. To start the server, run `npm
```

Comment:
Did we remove docker-compose from boltzmann scaffolding? Should we re-add it, or is it an opinion too far?

Reply:
A-ha, I remember now. We generated `docker-compose.yml` back when boltzmann was generated with make-it-eazy & lost it when we rewrote boltzmann as a standalone CLI.